### PR TITLE
Update and rename devops-command-center.rst to devops-command-center.md

### PR DESCRIPTION
**Pull request by justinegeffen**
Moving content from repo to admin guide.

### CODE REVIEW

**Review by amyblais — APPROVED** [review]

**Review by agarciamontoro — APPROVED** [review]
:tada:

File: source/administration/devops-command-center.rst [hunk 1]
```diff
@@ -1,7 +1,177 @@
...
+
+.. contents::
+  :depth: 2
+  :local:
+  :backlinks: entry
+  
+API Documentation
+-----------------
+
+The Mattermost Incident Management API specification is available `here <https://github.com/mattermost/mattermost-plugin-incident-management/blob/master/server/api/api.yaml>`_.
```
Comment (justinegeffen) [inline source/administration/devops-command-center.rst:24]: @agarciamontoro, is there any more information we can add here? Maybe a link to any help wanted tickets?

Comment (agarciamontoro) [reply source/administration/devops-command-center.rst:24]: I think it's ok for now. Maybe we could say that the documentation is a work in progress? Up to you!

**Comment by justinegeffen** [comment]
Merging this so that it can be iterated on for next release.

**Pull request merged**